Dockerfile

```
@@ -18,36 +18,47 @@ RUN composer update \
 FROM php:8.0-cli-alpine as compile
 
 ENV PHP_ZSTD_VERSION="master"
+ENV PHP_BROTLI_VERSION="4504e4186e79b197cfcb75d4d09aa47ef7d92fe"
 
 RUN apk add --no-cache \
     git \
     autoconf \
     make \
     g++  \
-    zstd-dev
+    zstd-dev \
+    brotli-dev
 
 ## Zstandard Extension
 FROM compile AS zstd
-RUN git clone --recursive --depth 1 --branch $PHP_ZSTD_VERSION https://github.com/kjdev/php-ext-zstd.git \
+RUN git clone --branch $PHP_ZSTD_VERSION https://github.com/kjdev/php-ext-zstd.git \
   && cd php-ext-zstd \
   && phpize \
   && ./configure --with-libzstd \
   && make && make install
 
+FROM compile as brotli
+RUN git clone --branch $PHP_BROTLI_VERSION https://github.com/kjdev/php-ext-brotli.git \
+ && cd php-ext-brotli \
```

Also, it looks like you can't checkout a commit sha like this. Let's update it to:

```suggestion
RUN git clone --recursive --depth 1 https://github.com/kjdev/php-ext-brotli.git \
 && cd php-ext-brotli \
 && git reset --hard $PHP_BROTLI_COMMIT \
```